### PR TITLE
Add redirection for organization profile

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds a simple redirection for https://software-gardening.github.io --to--> https://github.com/software-gardening . This is mostly for potential aesthetic and ease of sharing the organization location long-term.

### CODE REVIEW

**Review by falquaddoomi — APPROVED** [review]

**Review by vincerubinetti — APPROVED** [review]

**Review by gwaybio — APPROVED** [review]
FWIW, it might be worth specifying how many approvals by maintainers you need to merge items - for example, I don't think all three of us need to. (The waylab only requires one approval, FWIW)

**Comment by d33bs** [comment]
Thanks @falquaddoomi, @vincerubinetti , and @gwaybio for the reviews!

I took a quick look at enabling required 1 review per PR for all repositories in the organization and found that rulesets appear to be the quickest way to enable this. I created a ruleset for the organization, but it appears that we cannot enforce it without Enterprise access to GitHub. I'm going to explore the avenues to Enterprise access and alongside this what the impact might be of using branch rules at the repo level.

**Pull request merged**
